rpc_differ/rpc_differ.py

```diff
@@ -197,8 +197,20 @@ def make_rpc_report(repo_dir, old_commit, new_commit,
     rpc_repo_url = args.rpc_repo_url
     osa_differ.update_repo(repo_dir, rpc_repo_url, args.update)
 
-    # Are these commits valid?
-    osa_differ.validate_commits(repo_dir, [old_commit, new_commit])
+    # Are these commits valid? Just in case the commit is a
+    # PR ref, we try both the ref given and the ref prepended
+    # with the remote 'origin'.
```

Comment:
I'm not overly familiar with this code but it strikes me that https://github.com/rcbops/rpc_differ/pull/18/files#diff-7da6eeeb8f577172dfef67d1fbff0a3bL312 might be a better place to do this. The refs passed to `make_rpc_report` are used in other places, it looks like this allows for potential bugs due to different refs being used in different parts of the code.

You might also want to add a line to log when the ref is changed.

It might be worth creating a separate function for this to remove the code duplication.